update the flag documentation (#137)
FredTheDino/sylt-lang: README.md

Currently, Sylt can only run single files. The last filename given is
run.

> The `-v` flag also lets you see a lot of debug output. If you want

Reviewer: ```suggestion
The `-v` flag also lets you see some debug output. If you want
```

to debug the compiler and runtime this might be helpful.

> The `-vv` flag gives even more debug output. Don't count seeing anything
> from your own program!

Reviewer: ```suggestion
The `-vv` flag gives even more debug output. Don't count on seeing anything
from your own program!
```
